.circleci/config.yml

```diff
@@ -370,6 +371,58 @@ jobs:
             cd _build/release && ctest -j 16 -VV --output-on-failure
           no_output_timeout: 1h
 
+  linux-gcs-build:
```

Comment:
FYI that https://github.com/facebookincubator/velox/pull/1427 will likely land before this. We should then extend `linux-parquet-hive-storage-adapters-build` run to test CGS as well. This works for now.